projects/igniteui-angular/src/lib/data-operations/data-util.ts

```diff
@@ -30,24 +30,31 @@ export enum DataType {
  * @hidden
  */
 export class DataUtil {
-    public static sort<T>(data: T[], expressions: ISortingExpression[], sorting: IgxSorting = new IgxSorting()): T[] {
+    public static sort<T>(data: T[], expressions: ISortingExpression[], sorting?: IGridSortingStrategy): T[] {
+        if (!sorting) {
+            sorting = new IgxSorting();
+        }
```

Comment:
Provide a default value in the definition: `new IgxSorting()`.

Reply:
Currently we invoke the **sort** method by always passing the sorting parameter, which could be null or undefined. If it is null or undefined, it will overwrite the default value, which we do not want to happen in this case. This is why I have currently extracted the null check inside the method itself.

Reply:
I stand corrected, the behavior has been updated.